File: src/api/src/main/java/fr/iut/uca/model/issues/Comment.java
package fr.iut.uca.model.issues;

import java.util.Date;

public class Comment {

    private final Date createdAt;

    private String author;

    private String content;

    public Comment(Date createdAt, String author, String content) {
        this.createdAt = createdAt;
        this.author = author;
        this.content = content;
    }

    public Date getCreatedAt() {
        return createdAt;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: src/app/model/issues/Comment.ts
export class Comment {
  private _createdAt: Date
  private _author: string
  private _content: string

  constructor(createdAt: Date, author: string, content: string) {
    this._createdAt = createdAt;
    this._author = author;
    this._content = content;
  }

  get createdAt(): Date {
    return this._createdAt;
  }

  set createdAt(value: Date) {
    this._createdAt = value;
  }

  get author(): string {
    return this._author;
  }

  set author(value: string) {
    this._author = value;
  }

  get content(): string {
    return this._content;
  }

  set content(value: string) {
    this._content = value;
  }
}
